modules/codegen/src/smithy4s/codegen/Renderer.scala

```diff
@@ -331,13 +331,20 @@ private[codegen] class Renderer(compilationUnit: CompilationUnit) { self =>
       originalName: String,
       fields: List[Field],
       recursive: Boolean,
-      hints: List[Hint]
+      hints: List[Hint],
+      extendsType: Option[String] = None,
```

Comment:
rename field to `adtParent`, because we should distinguish between that and the future cases for mixins